TypeCobol/Compiler/CodeElements/Expressions/Expression.cs

```diff
@@ -121,6 +147,16 @@ public class ClassCondition : ConditionalExpression
 
 		public SyntaxProperty<bool> InvertResult { get; private set; }
 
+        public override object NodeData()
+        {
+            return new object[] { DataItem, CharacterClassNameReference, DataItemContentType, InvertResult };
```

Comment:
I'm not sure about that, using arrays makes it even harder for the caller to use the property. Maybe we should switch to a `string` property ? We need to discuss this with @mayanje.

Reply:
I agree that arrays are not a good solution as it requires to know the NodeType to be able to read it.
It could be easier to use as a string.

Reply:
I think we could simply call it Data or Items or DataItems not NodeData, and assume that it is always an array of object which is returned